src/core/model.js

```diff
@@ -1,12 +1,11 @@
-import { hasValueInKey } from '../util/object'
-
 export const State = {
   FETCHING: 'fetching',
   FETCHED: 'fetched',
   ERROR: 'error'
 }
 
 // checking value
-export const isFetchingSlot = slot => hasValueInKey(slot, 'state', State.FETCHING)
-export const isFetchedSlot = slot => hasValueInKey(slot, 'state', State.FETCHED)
-export const isErrorSlot = slot => hasValueInKey(slot, 'state', State.ERROR)
+const stateIs = value => obj => !!obj && obj.state === value
+export const isFetchingSlot = stateIs(State.FETCHING)
+export const isFetchedSlot = stateIs(State.FETCHED)
+export const isErrorSlot = stateIs(State.ERROR)
```

Comment:
Maybe we can now remove the `Slot` suffix now that they don't receive arguments? Also, have you considered `isFetching`, `hasFetched` and `hasErrors`? They make more sense gramatically.